src/lib/layout/Searchbar.tsx

```diff
@@ -225,14 +225,17 @@ const getNextCursor = (
 const getPlaceholder = ({
   isWasm,
   isPool,
+  isMove,
 }: {
   isWasm: boolean;
   isPool: boolean;
+  isMove: boolean;
 }) => {
   const wasmText = isWasm ? "/ Code ID / Contract Address" : "";
   const poolText = isPool ? "/ Pool ID" : "";
+  const moveText = isMove ? "/ Module" : "";
```

Comment:
Add space before each of them
" / Code ID..." : ""
" / Pool ID..." : ""
" / Module..." : ""